src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,27 @@
+import { useReactiveVar } from '@apollo/client';
+import MuiAlert, { AlertProps } from '@mui/material/Alert';
+import React from 'react';
+
+import { Container } from './Notification.styles';
+import { notificationService } from './NotificationSerivice';
+
+const Alert = React.forwardRef<HTMLDivElement, AlertProps>(function Alert(
+  props,
+  ref
+) {
+  return <MuiAlert elevation={10} ref={ref} {...props} />;
+});
+
+export const Notification = () => {
+  const alerts = useReactiveVar(notificationService.alerts);
+
+  return (
+    <Container>
+      {alerts.map(({ id, type, message }) => (
+        <Alert key={id} severity={type}>
```

Comment:
![image](https://user-images.githubusercontent.com/108259301/217781503-bd0f2eec-b198-4224-be4e-aa1314a33af0.png)
prints key error in the console when I fastly try to log in with invalid credentials multiple times. Maybe it would be better to use for example unique identifiers library, such as uuid, for keys if such situations occur